cms/djangoapps/contentstore/config/waffle_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Util methods for Waffle checks"""
+
+from cms.djangoapps.contentstore.config.waffle import waffle, ENABLE_CHECKLISTS_PAGE, ENABLE_CHECKLISTS_QUALITY
+# from courseware.access import has_access
+from student.roles import GlobalStaff
+
+
+def should_show_checklists_page(requesting_user):
+    """
+        Determine if the COURSE_HEALTH_PAGE waffle is set
+        and if the user is able to see it
+    """
+
+    if waffle().is_enabled(ENABLE_CHECKLISTS_PAGE):
+        if GlobalStaff().has_user(requesting_user):
+            return True
+
+    return False
+
+
+def should_show_checklists_quality(requesting_user, course_key):
+    """
+        Determine if the COURSE_HEALTH_PAGE waffle is set
```

Comment:
Nit: `COURSE_HEALTH_PAGE` waffle switch was renamed to `ENABLE_CHECKLISTS_QUALITY `, and maybe specify waffle switch instead of just waffle.